providers/common/include/prov/securitycheck.h

```diff
@@ -29,3 +35,9 @@ int ossl_digest_get_approved_nid(const EVP_MD *md);
 int ossl_digest_rsa_sign_get_md_nid(const EVP_MD *md);
 int ossl_securitycheck_enabled(OSSL_LIB_CTX *libctx);
 int ossl_tls1_prf_ems_check_enabled(OSSL_LIB_CTX *libctx);
+int ossl_hkdf_key_check_enabled(OSSL_LIB_CTX *libctx);
```

Comment:
Maybe remove these functions..
The only place they are used is within an #ifdef FIPS_MODULE block
so they may as well just use the fipscommon.h entries.

Reply:
Okay.

By the way, it seems that the `ossl_tls1_prf_ems_check_enabled` function is also only used within `#ifdef FIPS_MODULE` block.

Should I remove it together and use the `FIPS_tls_prf_ems_check` function?

Reply:
might as well.. I was doing it initially because the security_check functions were running in default and fips. Now that they are fips specific there is no need.

Reply:
The related functions removed.